nipype/info.py

```diff
@@ -99,7 +99,7 @@ def get_nipype_gitversion():
 
 # versions
 NIBABEL_MIN_VERSION = "2.1.0"
-NETWORKX_MIN_VERSION = "1.9"
+NETWORKX_MIN_VERSION = "2.0"
```

Comment:
And remove L146 with `pydotplus`?

Reply:
wait, so we are dropping both pydot and pydotplus?